tests/ui/asm/aarch64/type-f16.rs

```diff
@@ -0,0 +1,22 @@
+//@ only-aarch64
+
+#![feature(f16, f128)]
+use std::arch::asm;
+#[inline(never)]
+pub fn f32_to_f16(a: f32) -> f16 {
```

Comment:
You can remove this function, it was just to show what the assembly should look like